src/main/java/org/breedinginsight/brapps/importer/services/processors/experiment/create/model/PendingImportObjectData.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.breedinginsight.brapps.importer.services.processors.experiment.create.model;
+
+import lombok.*;
+import org.brapi.v2.model.core.BrAPIStudy;
+import org.brapi.v2.model.core.BrAPITrial;
+import org.brapi.v2.model.pheno.BrAPIObservationUnit;
+import org.breedinginsight.brapps.importer.model.response.PendingImportObject;
+
+@Getter
+@Setter
+@Builder
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+public class PendingImportObjectData {
```

Comment:
Ok, just wondering if the class name should be specific to experiment to prepare for that possibility.

Reply:
It could be, it's already under the experiment package namespace so I guess it's a matter of preference. Could we leave it as-is until the point we may do some Germplasm or other processor work?